### DISCUSSION TITLE
Is Canva Pro still offered in the student account offers?

### DISCUSSION THREAD

**Discussion by melissalhobbs** [community/community#28136] [GitHub Education] [answered]
I was researching online for free canva pro for students, and it said that it is within Github's student pack, but I haven't found it anywhere... Does anyone know what's going on?

**Comment by Gisgar3** [comment]
I just looked on the [GitHub Student Developer Pack](https://education.github.com/pack?sort=az) site to see what they offer for students and Canva Pro doesn't seem to be on there. If I remember correctly it used to be awhile back but it *may no longer* be offered ☹️

**Reply by khivraj-sankhla** [reply]
Ydy

**Reply by suman4576** [reply]
same problem

**Reply by danny7953** [reply]
any solution or we cant do it anymore??

**Reply by oumaimamajdoub** [reply]
même problème

**Reply by samyakjain1908** [reply]
Canva Pro benefit is no longer available in Github Student Developer pack

**Reply by aodh711** [reply]
I need someone to program an online store for me

**Reply by ProgrammerJayT** [reply]
@aodh711 Sure, drop me an email theanthem8@gmail.com

**Reply by hitenmuniyal** [reply]
@aodh711 send me your requirements on 11hiten.muniyal@gmail.com

**Comment by IngridLazar** [comment]
try to log in on "canva for students" with your github account

**Reply by Vfposselt** [reply]
According to their FAQ, "Canva for Education is not currently available for college and university students, but you can still use Canva free or Pro".

https://www.canva.com/pro/

**Comment by bassa846** [comment, marked as answer]
Thank you for contacting GitHub Education about the Canva benefit. The Canva benefit is no longer offered in the GitHub Student Developer Pack.
For students who are already on the 12 month benefit, you will continue to have access for the remainder of your 12 month offer period & can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.

**Reply by abdulrahman-hegazy** [reply]
Hi @bassa846  can you help me over here
https://github.com/orgs/community/discussions/28390
It would be a life changing opportunity

**Reply by bassa846** [reply]
@Abdelrhman161 please submit a support ticket so someone from our team can help troubleshoot. 
https://support.github.com/contact/education

**Reply by abdulrahman-hegazy** [reply]
why the ticket is closed, directly when i click submit.

![15373735637](https://user-images.githubusercontent.com/105675263/185970278-c3a76bf6-d536-4803-9452-62260e943886.jpg)

**Comment by kil0delt4** [comment]
Hey @bassa846, I have claimed the canva pro on my student developer pack and I have been using it for 3-4 months... but today suddenly I got downgraded to free version. Although you have mentioned above that those with access can use for 12 months. So, a little help here...

**Reply by AyeshaChoudhari7** [reply]
But I have access still

**Reply by kil0delt4** [reply]
That's the issue... I don't have it ☹️

**Reply by mmvarma-pro** [reply]
Same

**Reply by Fleurpunt** [reply]
same, what can I do to get it back?

**Reply by Tibaoh** [reply]
Je n'y ai plus accès non plus alors que j'étais censé y avoir accès jusqu'en février

**Reply by alicealtafin** [reply]
Perdi o meu acesso também, usava o canva pro a um ano pela GitHub e hoje ele saiu automaticamente. Como posso reivindicar ele por mais 12 meses como citado anteriormente aqui?

**Reply by garcianaty** [reply]
> Perdi o meu acesso também, usava o canva pro a um ano pela GitHub e hoje ele saiu automaticamente. Como posso reivindicar ele por mais 12 meses como citado anteriormente aqui?

Pelo que percebi por já não conseguimos ter acesso, o Canva já não faz parte do pack estudante.

**Comment by bassa846** [comment]
Canva has been alerted about your access issues, please contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) regarding this offer. They will have to troubleshoot your access for you.

**Reply by Sajjad5322** [reply]
I was just because of Canva now it's not here so sad 😢

**Reply by aleixomaduu** [reply]
I used to make all my presentations on Canva, for my professors and my students....... I'm so sad...... :(

**Comment by absafwan** [comment]
I am logging github for canva pro but its not available 😢 very bad  @github-staff

**Comment by Mahmudatulaulia** [comment]
hello, permission to ask..
my canva account was previously connected to github, but a few days ago my pro account couldn't be used anymore,,
anyone know why?
is the limit of using a Canva pro account only 1 year?

**Reply by samus-aran** [reply]
Hi @Mahmudatulaulia  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. For students who have already claimed for their 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions. Hope this helps.

**Reply by kil0delt4** [reply]
> Hi @Mahmudatulaulia 👋 ! Thanks for the great question about Global Campus Benefits.
> 
> We no longer provide the Canva benefit as part of the Student Developer Pack. For students who have already claimed for their 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions. Hope this helps.

Well i have tried contacting canva support and they ask to contact github support team. It's like both of them are throwing us in circles!!

**Comment by kareembux** [comment]
How to download Canva pro

**Comment by raza20** [comment]
hi

**Comment by raza20** [comment]
hi

**Comment by heishubham** [comment]
It will impact in their user base in the years to come.

**Comment by yaya2devops** [comment]
This is so sad. Is there any alternatives.

**Reply by aodh711** [reply]
[canva pro](https://sahelcard.com)
